docs/examples/005-viz-with-holoviz.ipynb

```diff
@@ -0,0 +1,431 @@
+{
```

Comment:
<p>Add a paragraph that clarifies things like: "From now on, we have <code>GeoDataFrame</code> to be visualized with Holoviz tools, and there is actually nothing to do with UXarray."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>